dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/entity/Schedule.java

```diff
@@ -108,10 +108,21 @@ public class Schedule {
     private int userId;
 
     /**
-     * created user name
+     * operator user id
+     */
+    private Integer operator;
```

Comment:
It's better to set an explicit name like `modifyUserId`.

Reply:
> It's better to set an explicit name like `modifyUserId`.
This one was written with reference to the previous naming, and I thought it would be better to keep it uniform, such as the one below:
![image](https://github.com/user-attachments/assets/a9098224-20fe-4bfd-8b6e-0885f7f83281)

Reply:
These are problems left over from history and will be reconstructed in the future

Reply:
> These are problems left over from history and will be reconstructed in the future

modifications completed